### ISSUE TITLE
new-UDHeading - broken

### ISSUE THREAD

**Issue by Phyziik** [ironmansoftware/universal-dashboard#133]
Can't use it : 
![image](https://user-images.githubusercontent.com/37376232/39475335-df45fe34-4d25-11e8-8db2-72c8554ba81c.png)

Tried to move it around, content, endpoint.

**Comment by adamdriscoll** [comment]
@Phyziik Can you validate that you are running 1.6.0-beta4? I tried this with latest and it works for me.

**Comment by Phyziik** [comment]
Same error in 1.6.0-beta4

Debug dashboard with error : 
```
$homePage = New-UDPage -Name "Home" -Content {
   New-UDRow -Columns {
      New-UDColumn -Endpoint {
         New-UDHeading -Content {"Hello"} -Size 4   
      }
   }
} 
  
$testSelection = New-UDPage -Url "/Wizard/TestSelection" -Endpoint { 
   New-UDCard -Title "Tests selection" -Content {
      New-UDRow -Columns {

      }
   }
} 
  

$redirector = New-UDPage -Url "/ComputerMonitoring/:testString" -Endpoint {
   param($testString)
   New-UDCard -Title "Test Card" -Content {"$testString"} -Reveal "test"
}


$MyDashboard = New-UDDashboard -Title "Debugging environment" -Pages @($homePage,$testSelection) -EndpointInitializationScript {
   Enable-UDLogging -FilePath "c:\debug_UDlogger.txt" -Level Error -Console
}

Start-UDDashboard -Port 1000 -Dashboard $MyDashboard -AutoReload
```

**Comment by adamdriscoll** [comment]
Fixed in 1.6.0
